pkg/asset/ignition/bootstrap/common.go

```diff
@@ -671,6 +672,7 @@ func (a *Common) addParentFiles(dependencies asset.Parents) {
 		&tls.ServiceAccountKeyPair{},
 		&tls.JournalCertKey{},
 		&tls.IronicTLSCert{},
+		&tls.BMCVerifyCA{},
```

Comment:
Since you already have `BMCVerifyCA` in the PlatformData, instead of doing this you could just create a file at `data/data/bootstrap/baremetal/files/opt/openshift/bmc-ca/verify_ca.crt.template` with the content:

```
{{ .PlatformData.BareMetal.BMCVerifyCA }}
```

Then you wouldn't need the BMCVerifyCA asset at all, and its one line of code could be rolled straight into the BMCVerifyCAConfigMap asset.